src/Serilog.Sinks.AzureEventHub/Serilog.Sinks.AzureEventHub.csproj

```diff
@@ -3,7 +3,7 @@
 		<Description>Write Serilog events to Azure Event Hub</Description>
 		<VersionPrefix>4.1.0</VersionPrefix>
```

Comment:
Since the dependencies are changing entirely with this PR, I think we should revise the major version to 5.0.0.

Reply:
Good point! Thanks! 
As for (at least) some projects this might change the final set of dependencies it sounds like a good idea to bump version to let the users know. 
I have pushed additional commit 0cf99dc575e98d80e72bcd8871ad9704b7b968de with corresponding description to this PR.